src/EventHub/EventHub/Cmdlets/Namespace/SetAzureEventHubNamespace.cs

```diff
@@ -58,6 +58,8 @@ public class SetAzureEventHubNamespace : AzureEventHubsCmdletBase
         [Parameter(Mandatory = false, ParameterSetName = AutoInflateParameterSet, ValueFromPipelineByPropertyName = true, Position = 4, HelpMessage = "The eventhub throughput units.")]
         public int? SkuCapacity { get; set; }
 
+        public const string ChangeDesc = "'State' Parameter is being deprecated without being replaced";
+        [CmdletParameterBreakingChange("State", ChangeDescription = ChangeDesc)]
```

Comment:
breaking change is disallowed without upgrading major version, please remove the attribute CmdletParameterBreakingChange

Reply:
@erich-wang [this attribute](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletparameterbreakingchangeattribute) is used to notify the user that the parameter will be removed in an upcoming breaking change release along with the message that's provided in the `ChangeDescription` property. The `-State` parameter is still available for users to provide, so this change won't break any existing scripts 😀